CHANGELOG.md

```diff
@@ -104,6 +110,13 @@ of response and not the raw API endpoint response
     - Changed `sfEventSqsToDbRecords` Lambda to use new upsert helpers for executions, granules, and PDRs
     to ensure out-of-order writes are handled correctly when writing to Elasticsearch
   - **CUMULUS-2510**
+    - Updated `@cumulus/sfEventSqsToDbRecords/write-execution` to publish SNS
+      messages after a successful write to Postgres, DynamoDB, and ES.
+    - Updated `@cumulus/api/endpoints/executions` to publish an SNS message
```

Comment:
This is no longer true

Reply:
Ah good catch

Reply:
[`36eef2c` (#2422)](https://github.com/nasa/cumulus/pull/2422/commits/36eef2c1080a9bdefcb81119a18d230ca78266d5)